### DISCUSSION TITLE
ESP32 USE_ARDUINO_OTA

### DISCUSSION THREAD

**Discussion by Bournetoride** [arendst/Tasmota#18485] [Q&A] [answered]
After uncommenting 
// -- OTA -----------------------------------------
#define USE_ARDUINO_OTA                          // Add optional support for Arduino OTA (+13k code)
I get this error 
In function 'void ArduinoOtaLoop()':
tasmota/tasmota_support/support_tasmota.ino:1668:8: error: 'class MDNSResponder' has no member named 'update'

**Comment by barbudor** [comment]
Any specific reason you want support for Arduino OTA?
Doesn't neither of Tasmota OTA suits you?

**Comment by Bournetoride** [comment]
Because I'm rolling my own and want avoid
Sending invitation to 192.168.1.149 ..........
16:36:26 [ERROR]: No response from the ESP
*** [upload] Error 1

**Comment by Jason2866** [comment]
For Arduino OTA you have to enable mDNS

**Comment by Bournetoride** [comment]
OK so i enable mDNS (true)
then I get
tasmota_support/support_tasmota.ino:1668:8: error: 'class MDNSResponder' has no member named 'update'
although that line number is a little spurious as ArduinoOtaLoop does not start until line 1681

**Comment by Jason2866** [comment]
Ohh, probably no one tried ArduinoOTA ever with Tasmota with ESP32.

**Reply by MrDix** [reply]
I've run into the same problem. 
Will it become the status of a 'bug' which is going to be fixed or will ESP32 devices stay excluded from this feature?

**Reply by Jason2866** [reply]
Since this is the first feedback, about since Tasmota is running on ESP32, we probably not do the port. We are discussing internal to remove ArduinoOTA completly.

**Reply by MrDix** [reply]
OK, thank for the information - so I do not need to retry enabling ArduinoOTA in every new release.

**Reply by Jason2866** [reply, marked as answer]
We decided to keep ArduinoOTA for esp8266. There will be no port done for ESP32.
